src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const list = document.querySelectorAll('li');
+
+for (const item of list) {
+  if (item.children.length > 0 && item.firstElementChild.tagName === 'UL') {
+    const span = document.createElement('span');
+
+    item.prepend(span);
+    span.append(item.childNodes[1]);
+  }
+}
+
+tree.addEventListener('click', event => {
+  if (event.target.tagName === 'SPAN') {
+    const ulStyles = event.target.parentElement.lastElementChild.style;
+
+    if (ulStyles.display === 'none') {
+      ulStyles.display = 'block';
+    } else {
+      ulStyles.display = 'none';
+    }
```

Comment:
here instead of getting the styles and changing display of the list, you can utilize [hidden property](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden)